documentation/user_docs/docs/Download_and_setup.rst

```diff
@@ -1,311 +1,126 @@
-##################
-Download and setup
-##################
+####################
+ Download and setup
+####################
+
+*********************
+ System Requirements
+*********************
+
+-  Windows 64-bit, or Linux 64-bit operating system
 
+-  Plenty of free disk space in order to cope with the combined data files (> 30GB)
```

Comment:
I wouldn't say this is part of the requirements for downloading and setting up Horace